frame/fast-unstake/src/lib.rs

```diff
@@ -125,6 +125,9 @@ pub mod pallet {
 		/// The origin that can control this pallet.
 		type ControlOrigin: frame_support::traits::EnsureOrigin<Self::RuntimeOrigin>;
 
+		/// The access to staking functionality.
+		type Staking: StakingInterface<Balance = BalanceOf<Self>, AccountId = Self::AccountId>;
```

Comment:
```suggestion
		type StakingInterface: StakingInterface<Balance = BalanceOf<Self>, AccountId = Self::AccountId>;
```

To be in-line with how `pallet-nomination-pools` is using this.

Reply:
I actually think that `type Staking` is a better name for it. StakingInterface seems redundant. Done this on purpose to float the idea.

Reply:
I prefer keeping it consistent with `nomination-pools`. You can chose which. 

That being said, `type Foo: Foo` is a common pattern in Substrate and Rust in general.

Reply:
Yeah, I guess I'm just not a big fan of the word `Interface` popping up in a type/variable name. It does make sense in a trait, because that's basically `Interface`.

Yep, I have been planning to make the nomination-pools change once the idea has been floated an accepted.

I also like `Staking`, because we will be able to pass `Staking` (pallet) to it and it'll be `Staking = Staking` :)